### PR TITLE
WIP Use eslint --fix to replace ' with " in the addon.

### PR DESCRIPTION
**Pull request by fzzzy**

### CODE REVIEW

File: addon/src/lib/channels.js [hunk 1]
```diff
@@ -139,17 +139,15 @@ function createChannelForAddonId(name, addonId) {
...
+    log("createChannelForAddonId FAILED (not installed)", name, addonId);
     return null;
   }
   const { mozExtensionHostname } = policy;
-  log('createChannelForAddonId', name, addonId, mozExtensionHostname);
+  log("createChannelForAddonId", name, addonId, mozExtensionHostname);
 
   // Create the special about:blank URL for the extension.
   const baseURI = Services.io.newURI(
-    `moz-extension://${mozExtensionHostname}/_blank.html`,
```
Comment (lmorchard) [inline addon/src/lib/channels.js]: May be a harmless change, worth omitting

File: addon/src/lib/prefs.js [hunk 2]
```diff
@@ -3,41 +3,41 @@
...
 
-const PREF_BRANCH = 'testpilot.';
-const ENV_PREF = 'testpilot.env';
+const PREF_BRANCH = "testpilot.";
+const ENV_PREF = "testpilot.env";
 
 const prefObserver = {
   register() {
-    Services.prefs.addObserver(PREF_BRANCH, this, false);
+    Services.prefs.addObserver(PREF_BRANCH, this);
```
Comment (lmorchard) [inline addon/src/lib/prefs.js]: May be a harmless change, worth omitting

File: addon/src/lib/prefs.js [hunk 3]
```diff
@@ -3,41 +3,41 @@
...
   },
   unregister() {
-    Services.prefs.removeObserver(PREF_BRANCH, this, false);
+    Services.prefs.removeObserver(PREF_BRANCH, this);
```
Comment (lmorchard) [inline addon/src/lib/prefs.js]: May be a harmless change, worth omitting

File: addon/src/lib/appStartup.js [hunk 4]
```diff
@@ -18,10 +18,10 @@ export function startupDone() {
 
 export const startupObserver = {
   register() {
-    Services.obs.addObserver(this, 'sessionstore-windows-restored', false);
```
Comment (lmorchard) [inline addon/src/lib/appStartup.js:21]: May be a harmless change, worth omitting

File: addon/src/lib/appStartup.js [hunk 5]
```diff
@@ -18,10 +18,10 @@ export function startupDone() {
...
+    Services.obs.addObserver(this, "sessionstore-windows-restored");
   },
   unregister() {
-    Services.obs.removeObserver(this, 'sessionstore-windows-restored', false);
```
Comment (lmorchard) [inline addon/src/lib/appStartup.js]: May be a harmless change, worth omitting

**Review by lmorchard — APPROVED** [review]
Okay. So I scrolled through this twice forwards and once backwards. Found a few spots that weren't just quote changes - possibly harmless, maybe not worth the chance. Seems :+1: to me, could use a going through by someone not me.

File: addon/src/lib/appStartup.js (hunk 4 above)
Comment (fzzzy) [reply addon/src/lib/appStartup.js:21]: I fixed this, but github isn't marking it as outdated!

**Comment by ghost** [comment]
Looks good to me too

File: addon/src/lib/appStartup.js (hunk 4 above)
Comment (meandavejustice) [reply addon/src/lib/appStartup.js:21]: prolly cus there is still a change on this line

**Pull request merged**